index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Tribute page</title>
+</head>
+<body>
+<main id ="main">
+<h1 id = "title" >President Barack Obama</h1>
+</div>
+<div id="img-div">
+<img style src="https://www.whitehouse.gov/wp-content/uploads/2021/01/44_barack_obama.jpg?resize=640,640" alt="American president in white house" >
+<div id = "img-caption">
+<h2>Barack Obama served as the 44th President of the United States. His story is the American story — values from the
+     heartland, a middle-class upbringing in a strong family, hard work and education as the means of getting ahead, 
+     and the conviction that a life so blessed should be lived in service to others.</h2>
+</div>
+
+</div>
```

Comment:
I can see the use of so many divs in your code please kindly avoid them. 
For accessibility's sake, try and implement the use of semantic tags such as <section>

Reply:
Great job pointing out the error and stating the reason.
[OPTIONAL] I think it will be clearer if you complete your statement about the semantic tags that the writer can use. 😊 

[OPTIONAL] I think you can make use of emojis to make the code review more human-friendly 👽🤖.